spec/integration/hanami/controller/cache_spec.rb

```diff
@@ -160,6 +161,15 @@ def call(_params)
       fresh etag: 'updated', last_modified: Time.now
     end
   end
+
+  class NilValue
+    include Hanami::Action
+    include Hanami::Action::Cache
+
+    def call(_params)
+      fresh etag: nil, last_modified: nil
```

Comment:
Can you separate this test? I prefer in one action use `etag: nil` and in other action `last_modified: nil`.